tensorflow/lite/delegates/webnn/BUILD

```diff
@@ -31,10 +31,26 @@ cc_library(
         "//tensorflow/lite/c:common",
         "//tensorflow/lite/schema:schema_fbs",
         "//tensorflow/lite/tools/optimize/sparsity:format_converter",
+        "@webnn_native_project//:webnn-native",
         "@FP16",
     ],
 )
 
+cc_binary(
+    name = "webnn_external_delegate.so",
+    srcs = [
+        "webnn_delegate_adaptor.cc",
+    ],
+    linkshared = 1,
+    linkstatic = 1,
```

Comment:
Why does it set both `linkshared` and `linkstatic`?

Reply:
I just refer to the guideline at https://www.tensorflow.org/lite/performance/implementing_delegate#option_2_leverage_external_delegate, not quite sure the main reason why it use both here. But `linkstatic = 1` is default value, we can remove it. 

According to the bazel doc, it says: 
> If you specify both linkstatic=1 and linkshared=True, you get a single, mostly self-contained unit.